pkg/build/builder/util/consts.go

```diff
@@ -24,6 +24,8 @@ const (
 	StatusMessagePushImageToRegistryFailed       = "Failed to push the image to the registry."
 	StatusMessagePullBuilderImageFailed          = "Failed pulling builder image."
 	StatusMessageFetchSourceFailed               = "Failed to fetch the input source."
+	StatusMessageFetchImageContentFailed         = "Failed to extract image content."
+	StatusMessageManageDockerfileFailed          = "Failed to prepare the dockerfile for the build."
```

Comment:
i'm a little surprised we're not including the error text in these messages, but i guess they should show up in LogSnippet, and you seem to be following the precedence we've established of not doing it (maybe we felt the errors were too raw to include in the message text)

Reply:
+1